_data/software/numpy.yml

```diff
@@ -0,0 +1,32 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: 'NumPy is the fundamental package for scientific computing with Python. '
```

Comment:
Think the trailing space causes this one to get quoted

Reply:
Looks like it doesn't come out with a quote? I checked, and it's all good.